src/server.ts

```diff
@@ -22,3 +22,7 @@ connectToDatabase()
     console.error('Database connection failed', error)
     process.exit()
   })
+
+function foo () {
```

Comment:
Was this a sample function you were testing? This can be removed

Reply:
Whoops, yep. Removed.